Cannot show upstream icon for git repository with no remote

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

For local repositories that I do not have connected to any remote, I'd like to show the git folder icon. Based on the documentation ("icon/text to display when the upstream is not known/mapped"), I expect this to work but it does not.
```
"fetch_upstream_icon": true,
"git_icon": "\ue5fb ", // leaving this line off to get the default does not work, either
"github_icon": "\uf408 ",
```
I have also tried this and it also doesn't work.
```
"fetch_upstream_icon": true,
"upstream_icons": {
  "github": "\uf408 ",
  "": "\ue5fb "
},
```

Not sure if this is a bug or working as designed. From the documentation, I would expect a folder with no remote to be "upstream is not mapped", so that sounds like a bug. I am suspecting that OMP expects there to be some kind of remote set and `git_icon` is used when it is "not known" (but then I don't know what "not mapped" would mean), in which case this may be by design.

### Theme

https://github.com/benallred/configs/blob/master/powershell/ben.omp.json

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 28.3.1

Shell: shell (7.5.4)

Prompt:

εé░ ∩É« εé░ 0ms εé░ 11:22:06 anchor123                                                                                                                                      Oh My Posh upgrade available (28.3.1 -> 28.5.0)
εé░ OneDrive εé░ Work\work-configs εé░
 ≤░ÿ¼ master εé▒ 

Segments:

ConsoleTitle(true)                         -   0 ms
Status(true)                               -   1 ms
Executiontime(true)                        -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Upgrade(true)                              -   0 ms
Path(true)                                 -   2 ms
Path(true)                                 -   2 ms
Path(false)                                -   2 ms
Dotnet(false)                              -   0 ms
Node(false)                                -   0 ms
Angular(false)                             -   0 ms
Terraform(false)                           -  11 ms
FileTypes(false)                           -   0 ms
Battery(false)                             -  21 ms
Text(false)                                -   0 ms
Git(true)                                  -   5 ms

Run duration: 55.7741ms

Cache path: C:\Users\ben.allred\AppData\Local\Packages\ohmyposh.cli_96v55e8n804z4\LocalCache\Local\oh-my-posh

Config path: C:\BenLocal\git\configs\powershell\ben.omp.json

Logs:

[DEBUG] 11:22:06.314 debug.go:39 ΓåÆ logging enabled, raw mode: true
[DEBUG] 11:22:06.318 store.go:]:141 ΓåÆ (session) key not found: prompt_count_cache
[TRACE] 11:22:06.318 store.go:](session prompt_count_cache) - 0s
[TRACE] 11:22:06.318 terminal.go:setPromptCount() - 0s
[TRACE] 11:22:06.318 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.318 terminal.go:setPwd:102 ΓåÆ C:\Users\ben.allred\OneDrive\Work\work-configs
[TRACE] 11:22:06.318 terminal.go:setPwd() - 0s
[TRACE] 11:22:06.318 terminal.go:Init() - 0s
[TRACE] 11:22:06.318 init.go:SessionID() - 0s
[DEBUG] 11:22:06.318 path_windows.go:PackageFamilyName:49 ΓåÆ PackageFamilyName: ohmyposh.cli_96v55e8n804z4
[TRACE] 11:22:06.318 path_windows.go:PackageFamilyName() - 0s
[TRACE] 11:22:06.318 path.go:Path() - 0s
[DEBUG] 11:22:06.334 store.go:init:94 ΓåÆ (session) loading template_cache
[DEBUG] 11:22:06.334 store.go:init:94 ΓåÆ (session) loading CONFIG
[DEBUG] 11:22:06.334 store.go:init:94 ΓåÆ (session) loading CONFIG_SOURCE
[DEBUG] 11:22:06.334 store.go:init:94 ΓåÆ (session) loading prompt_count_cache
[DEBUG] 11:22:06.334 store.go:init:94 ΓåÆ (session) loading toggle_cache
[TRACE] 11:22:06.335 store.go:init(session pwsh.76de1f99-0d06-435a-89e9-7c215915b5a3.omp.cache) - 16.9404ms
[TRACE] 11:22:06.335 path.go:Path() - 0s
[DEBUG] 11:22:06.338 store.go:init:94 ΓåÆ (device) loading upgrade_segment
[DEBUG] 11:22:06.338 store.go:init:94 ΓåÆ (device) loading reload
[DEBUG] 11:22:06.338 store.go:init:94 ΓåÆ (device) loading INITVERSIONPWSH
[DEBUG] 11:22:06.338 store.go:init:94 ΓåÆ (device) loading DSC_CONFIGURATION
[DEBUG] 11:22:06.338 store.go:init:94 ΓåÆ (device) loading DSC_SHELL
[TRACE] 11:22:06.338 store.go:init(device omp.cache) - 3.4155ms
[DEBUG] 11:22:06.338 store.go:]:154 ΓåÆ (device) found entry: reload - true
[TRACE] 11:22:06.338 store.go:](device reload) - 0s
[DEBUG] 11:22:06.338 gob.go:Get:29 ΓåÆ reload mode enabled
[DEBUG] 11:22:06.338 store.go:]:154 ΓåÆ (session) found entry: CONFIG_SOURCE - C:\BenLocal\git\configs\powershell\ben.omp.json
[TRACE] 11:22:06.338 store.go:](session CONFIG_SOURCE) - 0s
[DEBUG] 11:22:06.338 load.go:isTheme:354 ΓåÆ C:\BenLocal\git\configs\powershell\ben.omp.json is not a theme
[TRACE] 11:22:06.338 load.go:resolveConfigLocation() - 0s
[DEBUG] 11:22:06.338 store.go:]:154 ΓåÆ (device) found entry: DSC_CONFIGURATION - [0xc0000ee780]
[TRACE] 11:22:06.338 store.go:](device DSC_CONFIGURATION) - 0s
[DEBUG] 11:22:06.338 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.338 resource.go:Add:42 ΓåÆ item already exists
[TRACE] 11:22:06.342 load.go:readConfig() - 3.9965ms
[DEBUG] 11:22:06.342 store.go:]:177 ΓåÆ (device) setting entry: DSC_CONFIGURATION - [0xc0000ee780] with duration: infinite
[TRACE] 11:22:06.342 store.go:](device DSC_CONFIGURATION) - 0s
[TRACE] 11:22:06.342 load.go:parseConfigFile() - 3.9965ms
[DEBUG] 11:22:06.342 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.342 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.342 store.go:]:177 ΓåÆ (session) setting entry: toggle_cache - map[] with duration: infinite
[TRACE] 11:22:06.342 store.go:](session toggle_cache) - 0s
[TRACE] 11:22:06.342 load.go:Load() - 3.9965ms
[DEBUG] 11:22:06.342 store.go:]:177 ΓåÆ (session) setting entry: CONFIG_SOURCE - C:\BenLocal\git\configs\powershell\ben.omp.json with duration: infinite
[TRACE] 11:22:06.342 store.go:](session CONFIG_SOURCE) - 0s
[TRACE] 11:22:06.343 gob.go:Base64() - 519.3┬╡s
[DEBUG] 11:22:06.343 store.go:]:177 ΓåÆ (session) setting entry: CONFIG - /gH5fwMBAQZDb25maWcB/4AAAR8BB1BhbGV0dGUB/4IAAQtEZWJ1Z1Byb21wdAH/hAABA1ZhcgH/jgABCFBhbGV0dGVzAf+QAAEJVmFsaWRMaW5lAf+EAAEPU2Vjb25kYXJ5UHJvbXB0Af+EAAEPVHJhbnNpZW50UHJvbXB0Af+EAAEJRXJyb3JMaW5lAf+EAAEETWFwcwH/lAABB1VwZ3JhZGUB/5gAAQdFeHRlbmRzAQwAAQtBY2NlbnRDb2xvcgEMAAEUQ29uc29sZVRpdGxlVGVtcGxhdGUBDAABA1BXRAEMAAEGU291cmNlAQwAAQZGb3JtYXQBDAABElRlcm1pbmFsQmFja2dyb3VuZAEMAAEOVG9vbFRpcHNBY3Rpb24BDAABBkJsb2NrcwH/ngABBUN5Y2xlAf+iAAENSVRlcm1GZWF0dXJlcwH/pAABCFRvb2x0aXBzAf+cAAEHVmVyc2lvbgEEAAENTWlncmF0ZUdseXBocwECAAEFQXN5bmMBAgABEFNoZWxsSW50ZWdyYXRpb24BAgABCkZpbmFsU3BhY2UBAgABDVVwZ3JhZGVOb3RpY2UBAgABDlBhdGNoUHdzaEJsZWVkAQIAAQtBdXRvVXBncmFkZQECAAEXRW5hYmxlQ3Vyc29yUG9zaXRpb25pbmcBAgAAABf/gQQBAQdQYWxldHRlAf+CAAEMAQwAAP4CFf+DAwEBB1NlZ21lbnQB/4QAASIBB09wdGlvbnMB/4YAAQpQcm9wZXJ0aWVzAf+GAAEFQ2FjaGUB/4gAAQVBbGlhcwEMAAEOTGVhZGluZ0RpYW1vbmQBDAABD1RyYWlsaW5nRGlhbW9uZAEMAAEIVGVtcGxhdGUBDAABCkZvcmVncm91bmQBDAABDlRlbXBsYXRlc0xvZ2ljAQwAAQ9Qb3dlcmxpbmVTeW1ib2wBDAABCkJhY2tncm91bmQBDAABBkZpbGxlcgEMAAEEVHlwZQEMAAEFU3R5bGUBDAABFkxlYWRpbmdQb3dlcmxpbmVTeW1ib2wBDAABE0ZvcmVncm91bmRUZW1wbGF0ZXMB/4oAAQRUaXBzAf+MAAETQmFja2dyb3VuZFRlbXBsYXRlcwH/igABCVRlbXBsYXRlcwH/igABDkV4Y2x1ZGVGb2xkZXJzAf+MAAEOSW5jbHVkZUZvbGRlcnMB/4wAAQVOZWVkcwH/jAABB1RpbWVvdXQBBAABCE1heFdpZHRoAQQAAQhNaW5XaWR0aAEEAAEIRHVyYXRpb24BBAABCk5hbWVMZW5ndGgBBAABBUluZGV4AQQAAQtJbnRlcmFjdGl2ZQECAAEHRW5hYmxlZAECAAEHTmV3bGluZQECAAEPSW52ZXJ0UG93ZXJsaW5lAQIAAQVGb3JjZQECAAEHVG9nZ2xlZAECAAAAE/+FBAEBA01hcAH/hgABDAEQAAAt/4cDAQEFQ2FjaGUB/4gAAQIBCER1cmF0aW9uAQwAAQhTdHJhdGVneQEMAAAAEv+JAgEBBExpc3QB/4oAAQwAABb/iwIBAQhbXXN0cmluZwH/jAABDAAAJ/+NBAEBF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9Af+OAAEMARAAAC3/jwMBAQhQYWxldHRlcwH/kAABAgEETGlzdAH/kgABCFRlbXBsYXRlAQwAAAAp/5EEAQEYbWFwW3N0cmluZ11jb2xvci5QYWxldHRlAf+SAAEMAf+CAAA//5MDAQEGQ29uZmlnAf+UAAEDAQhVc2VyTmFtZQH/lgABCEhvc3ROYW1lAf+WAAEJU2hlbGxOYW1lAf+WAAAAE/+VBAEBA01hcAH/lgABDAEMAABc/5cDAQEGQ29uZmlnAf+YAAEGAQZTb3VyY2UBDAABCEludGVydmFsAQwAAQZMYXRlc3QBDAABBEF1dG8BAgABDURpc3BsYXlOb3RpY2UBAgABBUZvcmNlAQIAAAAe/50CAQEPW10qY29uZmlnLkJsb2NrAf+eAAH/mgAA/6r/mQMBAv+aAAEMAQRUeXBlAQwAAQlBbGlnbm1lbnQBDAABBkZpbGxlcgEMAAEIT3ZlcmZsb3cBDAABDkxlYWRpbmdEaWFtb25kAQwAAQ9UcmFpbGluZ0RpYW1vbmQBDAABCFNlZ21lbnRzAf+cAAEITWF4V2lkdGgBBAABCE1pbldpZHRoAQQAAQdOZXdsaW5lAQIAAQVGb3JjZQECAAEFSW5kZXgBBAAAACD/mwIBARFbXSpjb25maWcuU2VnbWVudAH/nAAB/4QAABT/oQIBAQVDeWNsZQH/ogAB/6AAACr/nwMBAv+gAAECAQpCYWNrZ3JvdW5kAQwAAQpGb3JlZ3JvdW5kAQwAAAAb/6MCAQENSVRlcm1GZWF0dXJlcwH/pAABDAAA/hWz/4ABDA1icmlnaHRfcHVycGxlByNjNTg2YzAKZ2l0OmJlaGluZAdwOmVycm9yFGdpdDphaGVhZF9hbmRfYmVoaW5kCHA6eWVsbG93B3N1Y2Nlc3MHIzhkYjA3YhJjb25zb2xlOmZvcmVncm91bmQHI2Q0ZDRkNBljb25zb2xlOmJyaWdodF9mb3JlZ3JvdW5kByNmZmZmZmYSY29uc29sZTpiYWNrZ3JvdW5kByMxZTFlMWUEY3lhbgcjNTY5Y2Q2CWhhbGZfY3lhbgcjMjg0ZTZiBnllbGxvdwcjZDdiYTdkCWdpdDphaGVhZAlwOnN1Y2Nlc3MFZXJyb3IHI2QxNjk2OQQHASABCXA6c3VjY2VzcwMLdHJhbnNwYXJlbnQAAQcGICDugrEgAAIHA++EqgEHcDplcnJvcgMLdHJhbnNwYXJlbnQAAgEDY2RuAQQxNjhoAAMLe3suRm9sZGVyfX0CL0M6XEJlbkxvY2FsXGdpdFxjb25maWdzXHBvd2Vyc2hlbGxcYmVuLm9tcC5qc29uAQRqc29uAwQBBnByb21wdAEEbGVmdAUEAQEOYWx3YXlzX2VuYWJsZWQEYm9vbAICAAEGLiB7eyBpZiBndCAuQ29kZSAwIH1975Gne3sgZWxzZSB9fe+Qrnt7IGVuZCB9fSABG3A6Y29uc29sZTpicmlnaHRfZm9yZWdyb3VuZAID7oKwAQlwOnN1Y2Nlc3MCBnN0YXR1cwEJcG93ZXJsaW5lBAEje3sgaWYgZ3QgLkNvZGUgMCB9fXA6ZXJyb3J7eyBlbmQgfX0AAQIOYWx3YXlzX2VuYWJsZWQEYm9vbAICAAEFc3R5bGUGc3RyaW5nDAgABmF1c3RpbgYUIHt7IC5Gb3JtYXR0ZWRNcyB9fSABFHA6Y29uc29sZTpiYWNrZ3JvdW5kAgPugrABFHA6Y29uc29sZTpmb3JlZ3JvdW5kAg1leGVjdXRpb250aW1lAQlwb3dlcmxpbmUAByMge3sgLkN1cnJlbnREYXRlIHwgZGF0ZSAuRm9ybWF0IH19IAEUcDpjb25zb2xlOmZvcmVncm91bmQDC3RyYW5zcGFyZW50AgR0aW1lAQVwbGFpbgAHCWFuY2hvcjEyMwEUcDpjb25zb2xlOmJhY2tncm91bmQFBHRleHQBBXBsYWluAAABBnByb21wdAEFcmlnaHQFAQc+T2ggTXkgUG9zaCB1cGdyYWRlIGF2YWlsYWJsZSAoe3sgLkN1cnJlbnQgfX0gLT4ge3sgLkxhdGVzdCB9fSkBCHA6eWVsbG93Awt0cmFuc3BhcmVudAIHdXBncmFkZQEFcGxhaW4AAAEGcHJvbXB0AQRsZWZ0BQMBAQVzdHlsZQZzdHJpbmcMBgAEZnVsbAcbcDpjb25zb2xlOmJyaWdodF9mb3JlZ3JvdW5kAQtmaXJzdF9tYXRjaAED7oKwAQtwOmhhbGZfY3lhbgIEcGF0aAEJcG93ZXJsaW5lBQNVe3sgaWYgcmVnZXhNYXRjaCAocmVwbGFjZSAiLyIgIlxcXFwiICJDOi9CZW5Mb2NhbC9naXQoLy4rKT8kIikgLlBhdGggfX0gZ2l0IHt7IGVuZCB9ff+je3sgaWYgcmVnZXhNYXRjaCAocmVwbGFjZSAiLyIgIlxcXFwiICJDOi9Xb3JrL3JlcG9zKC8uKyk/JCIpIC5QYXRoIH19IHt7LSBpZiAuRW52LldvcmtSZXBvc0xhYmVsIH19IHt7IC5FbnYuV29ya1JlcG9zTGFiZWwgfX0ge3sgZWxzZSB9fSByZXBvcyB7eyBlbmQgLX19IHt7IGVuZCB9fVV7eyBpZiByZWdleE1hdGNoIChyZXBsYWNlICIvIiAiXFxcXCIgIn4vT25lRHJpdmUoLy4rKT8kIikgLlBhdGggfX0gT25lRHJpdmUge3sgZW5kIH19AgMVQzovQmVuTG9jYWwvZ2l0KC8uKik/E0M6L1dvcmsvcmVwb3MoLy4qKT8Qfi9PbmVEcml2ZSgvLiopPwABAgVzdHlsZQZzdHJpbmcMBgAEZnVsbBBtYXBwZWRfbG9jYXRpb25zF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/45YAAMPQzovQmVuTG9jYWwvZ2l0BnN0cmluZwwDAAFcDUM6L1dvcmsvcmVwb3MGc3RyaW5nDAMAAVwTe3sgLkVudi5PbmVEcml2ZSB9fQZzdHJpbmcMAwABXAYuIHt7IHJlZ2V4UmVwbGFjZUFsbCAiXlxcXFwoLispIiAuUGF0aCAiJDEiIH19IAEbcDpjb25zb2xlOmJyaWdodF9mb3JlZ3JvdW5kAgPugrABBnA6Y3lhbgIEcGF0aAEJcG93ZXJsaW5lAAcwIHt7IGlmIC5TdGFja0NvdW50IH19e3sgLlN0YWNrQ291bnQgfX17eyBlbmQgfX0gARRwOmNvbnNvbGU6YmFja2dyb3VuZAID7oKwAQhwOnllbGxvdwIEcGF0aAEJcG93ZXJsaW5lAAMBAAEGcHJvbXB0AQRsZWZ0BQgBAQ1mZXRjaF92ZXJzaW9uBGJvb2wCAgAABjMgPCM4OTMxZmY+7p2/PC8+e3sgaWYgLkZ1bGwgfX0ge3sgLkZ1bGwgfX17eyBlbmQgfX0BFHA6Y29uc29sZTpmb3JlZ3JvdW5kAwt0cmFuc3BhcmVudAIGZG90bmV0AQVwbGFpbgABBg1mZXRjaF92ZXJzaW9uBGJvb2wCAgABFWZldGNoX3BhY2thZ2VfbWFuYWdlcgRib29sAgIAAQl5YXJuX2ljb24Gc3RyaW5nDBEADzwjMmM4ZWJiPu6apzwvPghucG1faWNvbgZzdHJpbmcMEQAPPCNmZjQzMDA+7pyePC8+EWNvbW1lbnRfeWFybl9pY29uBnN0cmluZwwFAAPumqcQY29tbWVudF9ucG1faWNvbgZzdHJpbmcMBQAD7pyeBk0ge3sgLlBhY2thZ2VNYW5hZ2VySWNvbiB9fSA8IzVmYTA0ZT7unJg8Lz4ge3sgaWYgLkZ1bGwgfX17eyAuRnVsbCB9fXt7IGVuZCB9fQEUcDpjb25zb2xlOmZvcmVncm91bmQDC3RyYW5zcGFyZW50AgRub2RlAQVwbGFpbgABAQ1mZXRjaF92ZXJzaW9uBGJvb2wCAgABBjMgPCNmZjQzMDA+7p2TPC8+IHt7IGlmIC5GdWxsIH19e3sgLkZ1bGwgfX17eyBlbmQgfX0BFHA6Y29uc29sZTpmb3JlZ3JvdW5kAwt0cmFuc3BhcmVudAIHYW5ndWxhcgEFcGxhaW4AAQENZmV0Y2hfdmVyc2lvbgRib29sAgIAAQZOIDwjYTY1OWZmPu6amjwvPiB7eyBpZiAuVmVyc2lvbiB9fXt7IC5WZXJzaW9uIH19IHt7IGVuZCB9fXt7IC5Xb3Jrc3BhY2VOYW1lIH19ARRwOmNvbnNvbGU6Zm9yZWdyb3VuZAMLdHJhbnNwYXJlbnQCCXRlcnJhZm9ybQEFcGxhaW4ABAlGaWxlVHlwZXMEFHA6Y29uc29sZTpmb3JlZ3JvdW5kAwt0cmFuc3BhcmVudAIEdGV4dAEFcGxhaW4FAT17eyAuRW52Lm9tcF9maWxlc19yZWFkbWUgfX17eyAuRW52Lm9tcF9maWxlc19kb2NrZXJjb21wb3NlIH19AAEJGGNvbW1lbnRfZGlzY2hhcmdpbmdfaWNvbgZzdHJpbmcMDAAKZjE3ZTQv87GfpBljb21tZW50X25vdF9jaGFyZ2luZ19pY29uBnN0cmluZwwMAApmMTdlOC/zsZ+oDWRpc3BsYXlfZXJyb3IEYm9vbAICAAEQZGlzY2hhcmdpbmdfaWNvbgZzdHJpbmcMBgAE87GfpA1jaGFyZ2luZ19pY29uBnN0cmluZwwGAATzsZ+mDGNoYXJnZWRfaWNvbgZzdHJpbmcMBgAE87GfohFub3RfY2hhcmdpbmdfaWNvbgZzdHJpbmcMBgAE87GfqBVjb21tZW50X2NoYXJnaW5nX2ljb24Gc3RyaW5nDAwACmYxN2U2L/Oxn6YUY29tbWVudF9jaGFyZ2VkX2ljb24Gc3RyaW5nDAwACmYxN2UyL/Oxn6IGbiB7eyBpZiBhbmQgKG5vdCAuRXJyb3IpIChsdCAuUGVyY2VudGFnZSAxMDApIH19e3sgLkljb24gfX08cDpjb25zb2xlOmZvcmVncm91bmQ+e3sgLlBlcmNlbnRhZ2UgfX0lPC8+e3sgZW5kIH19BAt0cmFuc3BhcmVudAIHYmF0dGVyeQEFcGxhaW4CBk17e2lmIGFuZCAoZXEgLlN0YXRlLlN0cmluZyAiRGlzY2hhcmdpbmciKSAobHQgLlBlcmNlbnRhZ2UgMzApfX1wOmVycm9ye3tlbmR9fTR7e2lmIGVxIC5TdGF0ZS5TdHJpbmcgIkRpc2NoYXJnaW5nIn19cDp5ZWxsb3d7e2VuZH19S3t7aWYgYW5kIChlcSAuU3RhdGUuU3RyaW5nICJDaGFyZ2luZyIpIChsdCAuUGVyY2VudGFnZSA2MCl9fXA6eWVsbG93e3tlbmR9fTJ7e2lmIGVxIC5TdGF0ZS5TdHJpbmcgIkNoYXJnaW5nIn19cDpzdWNjZXNze3tlbmR9fS57e2lmIGVxIC5TdGF0ZS5TdHJpbmcgIkZ1bGwifX1wOnN1Y2Nlc3N7e2VuZH19B3A6ZXJyb3IACBRwOmNvbnNvbGU6Zm9yZWdyb3VuZAELZmlyc3RfbWF0Y2gCC3RyYW5zcGFyZW50AgR0ZXh0AQVwbGFpbgUGMSB7eyBpZiAuU2VnbWVudHMuQ29udGFpbnMgIkRvdG5ldCIgfX3ugrF7eyBlbmQgfX0vIHt7IGlmIC5TZWdtZW50cy5Db250YWlucyAiTm9kZSIgfX3ugrF7eyBlbmQgfX0yIHt7IGlmIC5TZWdtZW50cy5Db250YWlucyAiQW5ndWxhciIgfX3ugrF7eyBlbmQgfX00IHt7IGlmIC5TZWdtZW50cy5Db250YWlucyAiVGVycmFmb3JtIiB9fe6CsXt7IGVuZCB9fTQge3sgaWYgLlNlZ21lbnRzLkNvbnRhaW5zICJGaWxlVHlwZXMiIH197oKxe3sgZW5kIH19MiB7eyBpZiAuU2VnbWVudHMuQ29udGFpbnMgIkJhdHRlcnkiIH197oKxe3sgZW5kIH19AAEYEWJyYW5jaF9haGVhZF9pY29uBnN0cmluZwwVABM8cDpnaXQ6YWhlYWQ+4oaRPC8+I3RlbXBsYXRlQ29tbWVudF9fYnJhbmNoX2JlaGluZF9pY29uBnN0cmluZwwFAAPihpMhdGVtcGxhdGVDb21tZW50X19icmFuY2hfZ29uZV9pY29uBnN0cmluZwwFAAPiiaIcdGVtcGxhdGVDb21tZW50X19jb21taXRfaWNvbgZzdHJpbmcMBgAE75CXIB50ZW1wbGF0ZUNvbW1lbnRfXy5IRUFEX19yZWJhc2UGc3RyaW5nDDUAM+6cqCDzsJisYnJhbmNoIG9udG8g87CYrG1hc3RlciAoMS8zKSBhdCDvkJcgYWJjMTIzNCB0ZW1wbGF0ZUNvbW1lbnRfXy5Xb3JraW5nLlN0cmluZwZzdHJpbmcMBwAFKzIgfjYZdGVtcGxhdGVDb21tZW50X19naXRfaWNvbgZzdHJpbmcMBgAE7pe7IBx0ZW1wbGF0ZUNvbW1lbnRfX2dpdGh1Yl9pY29uBnN0cmluZwwGAATvkIggFWJyYW5jaF9pZGVudGljYWxfaWNvbgZzdHJpbmcMEAAOPHA6Y3lhbj7iiaE8Lz4SYnJhbmNoX2JlaGluZF9pY29uBnN0cmluZwwWABQ8cDpnaXQ6YmVoaW5kPuKGkzwvPhBicmFuY2hfZ29uZV9pY29uBnN0cmluZwwZABc8cDpicmlnaHRfcHVycGxlPuKJojwvPgtjb21taXRfaWNvbgZzdHJpbmcMBgAE75CXIAh0YWdfaWNvbgZzdHJpbmcMBgAE75CSIBx0ZW1wbGF0ZUNvbW1lbnRfX2JyYW5jaF9pY29uBnN0cmluZwwMAApmMDYyYy/zsJisJnRlbXBsYXRlQ29tbWVudF9fYnJhbmNoX2lkZW50aWNhbF9pY29uBnN0cmluZwwFAAPiiaEgdGVtcGxhdGVDb21tZW50X18uU3RhZ2luZy5TdHJpbmcGc3RyaW5nDAQAAn4xBnNvdXJjZQZzdHJpbmcMBgAEcHdzaAticmFuY2hfaWNvbgZzdHJpbmcMFAASPHA6eWVsbG93PvOwmKw8Lz4gInRlbXBsYXRlQ29tbWVudF9fYnJhbmNoX2FoZWFkX2ljb24Gc3RyaW5nDAUAA+KGkR50ZW1wbGF0ZUNvbW1lbnRfXy5CcmFuY2hTdGF0dXMGc3RyaW5nDAUAA+KJoRR0ZW1wbGF0ZUNvbW1lbnRfX+6CsQZzdHJpbmcMBQAD7oKxDGZldGNoX3N0YXR1cwRib29sAgIAABl0ZW1wbGF0ZUNvbW1lbnRfX3RhZ19pY29uBnN0cmluZwwGAATvkJIgHnRlbXBsYXRlQ29tbWVudF9fLkhFQURfX25vcm1hbAZzdHJpbmcMDAAK87CYrG1hc3Rlcgb+AVQgPHA6Y29uc29sZTpmb3JlZ3JvdW5kPnt7IC5VcHN0cmVhbUljb24gfX08Lz57eyAuSEVBRCB9fXt7IGlmIC5CcmFuY2hTdGF0dXMgfX0ge3sgLkJyYW5jaFN0YXR1cyB9fXt7IGVuZCB9fXt7IGlmIC5Xb3JraW5nLkNoYW5nZWQgfX08cDpnaXQ6YmVoaW5kPiB7eyAuV29ya2luZy5TdHJpbmcgfX08Lz57eyBlbmQgfX17eyBpZiBhbmQgKC5TdGFnaW5nLkNoYW5nZWQpICguV29ya2luZy5DaGFuZ2VkKSB9fXt7IGVuZCB9fXt7IGlmIC5TdGFnaW5nLkNoYW5nZWQgfX08cDpnaXQ6YWhlYWQ+IHt7IC5TdGFnaW5nLlN0cmluZyB9fTwvPnt7IGVuZCB9fTxwOmNvbnNvbGU6Zm9yZWdyb3VuZD4g7oKxPC8+ARRwOmNvbnNvbGU6Zm9yZWdyb3VuZAMLdHJhbnNwYXJlbnQCA2dpdAEFcGxhaW4CA0h7eyBpZiBhbmQgKGd0IC5BaGVhZCAwKSAoZ3QgLkJlaGluZCAwKSB9fXA6Z2l0OmFoZWFkX2FuZF9iZWhpbmR7eyBlbmQgfX0oe3sgaWYgZ3QgLkFoZWFkIDAgfX1wOmdpdDphaGVhZHt7IGVuZCB9fSp7eyBpZiBndCAuQmVoaW5kIDAgfX1wOmdpdDpiZWhpbmR7eyBlbmQgfX0AAwEABAgEAQA= with duration: infinite
[TRACE] 11:22:06.343 store.go:](session CONFIG) - 0s
[TRACE] 11:22:06.343 gob.go:Store() - 519.3┬╡s
[TRACE] 11:22:06.343 gob.go:Get() - 4.5158ms
[TRACE] 11:22:06.343 terminal.go:Shell() - 0s
[TRACE] 11:22:06.343 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.343 store.go:]:154 ΓåÆ (session) found entry: template_cache - {map[Executiontime:0xc000258340 Git:0xc00025c000 Status:0xc0001e2870 Time:0xc0001e29b0 Upgrade:0xc0000b8900] map[] ~\OneDrive\Work\work-configs work-configs  ben.allred ballred-xps 7.5.4 pwsh C:\Users\ben.allred\OneDrive\Work\work-configs windows 28.3.1 29 0 0 0 false true}
[TRACE] 11:22:06.343 store.go:](session template_cache) - 0s
[TRACE] 11:22:06.343 cache.go:restoreCache() - 0s
[DEBUG] 11:22:06.343 writer.go:Init:107 ΓåÆ terminal program: Windows Terminal
[DEBUG] 11:22:06.343 writer.go:Init:108 ΓåÆ terminal shell: shell
[DEBUG] 11:22:06.343 terminal.go:Getenv:64 ΓåÆ NO DATA
[TRACE] 11:22:06.343 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[DEBUG] 11:22:06.343 store.go:]:141 ΓåÆ (device) key not found: accent_color
[TRACE] 11:22:06.343 store.go:](device accent_color) - 0s
[DEBUG] 11:22:06.343 terminal_windows.go:WindowsRegistryKeyValue:198 ΓåÆ ColorizationColor(DWORD): 0xC4459BAC
[TRACE] 11:22:06.343 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 11:22:06.343 colors_windows.go:GetAccentColor() - 0s
[TRACE] 11:22:06.343 colors.go:SetAccentColor() - 0s
[TRACE] 11:22:06.343 terminal.go:Shell() - 0s
[DEBUG] 11:22:06.343 terminal.go:Getenv:64 ΓåÆ 7.5.4
[TRACE] 11:22:06.343 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 11:22:06.344 debug.go:PrintDebug:24 ΓåÆ segment: Title
[DEBUG] 11:22:06.344 text.go:patchTemplate:149 ΓåÆ {{.Folder}}
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.344 terminal.go:Getenv:64 ΓåÆ 89
[TRACE] 11:22:06.344 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 11:22:06.344 terminal.go:Getenv:64 ΓåÆ 1
[TRACE] 11:22:06.344 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.344 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.344 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.344 segment.go:Execute:172 ΓåÆ segment: Text
[DEBUG] 11:22:06.344 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.344 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.344 segment.go:Execute:172 ΓåÆ segment: Status
[DEBUG] 11:22:06.344 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[DEBUG] 11:22:06.344 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.344 store.go:](session toggle_cache) - 0s
[TRACE] 11:22:06.344 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.344 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.344 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[TRACE] 11:22:06.344 terminal.go:StatusCodes() - 0s
[DEBUG] 11:22:06.344 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.344 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.344 segment.go:Execute:172 ΓåÆ segment: Executiontime
[DEBUG] 11:22:06.344 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.344 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.344 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.344 map.go:String:77 ΓåÆ status_template: {{ .Code }}
[DEBUG] 11:22:06.344 map.go:Bool:120 ΓåÆ always_enabled: true
[TRACE] 11:22:06.344 terminal.go:ExecutionTime() - 0s
[DEBUG] 11:22:06.344 map.go:Float64:127 ΓåÆ threshold: 500.000000
[DEBUG] 11:22:06.344 map.go:String:81 ΓåÆ style: austin
[DEBUG] 11:22:06.344 text.go:patchTemplate:149 ΓåÆ {{ .Code }}
[TRACE] 11:22:06.344 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.344 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.344 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.344 segment.go:Execute:172 ΓåÆ segment: Time
[DEBUG] 11:22:06.344 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.344 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.344 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.344 map.go:String:77 ΓåÆ time_format: 15:04:05
[DEBUG] 11:22:06.345 map.go:Bool:120 ΓåÆ always_enabled: true
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ  {{ if gt .Code 0 }}∩æº{{ else }}∩É«{{ end }} 
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ {{ if gt .Code 0 }}p:error{{ end }}
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ {{ if gt .Code 0 }}p:error{{ end }}
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ  {{ .Data.FormattedMs }} 
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ {{ if gt .Code 0 }}p:error{{ end }}
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ  {{ .Data.CurrentDate | date .Data.Format }} 
[TRACE] 11:22:06.345 terminal.go:Shell() - 0s
[TRACE] 11:22:06.345 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.345 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.345 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.345 segment.go:Execute:172 ΓåÆ segment: Upgrade
[DEBUG] 11:22:06.345 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.345 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.345 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.345 store.go:]:154 ΓåÆ (device) found entry: upgrade_segment - &{28.5.0 28.3.1}
[TRACE] 11:22:06.345 store.go:](device upgrade_segment) - 0s
[DEBUG] 11:22:06.345 text.go:patchTemplate:149 ΓåÆ Oh My Posh upgrade available ({{ .Data.Current }} -> {{ .Data.Latest }})
[DEBUG] 11:22:06.346 terminal_windows.go:TerminalWidth:99 ΓåÆ terminal width: 211
[TRACE] 11:22:06.346 terminal_windows.go:TerminalWidth() - 621.8┬╡s
[DEBUG] 11:22:06.346 engine.go:shouldFill:161 ΓåÆ no filler specified
[TRACE] 11:22:06.346 terminal.go:Shell() - 0s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 segment.go:Execute:172 ΓåÆ segment: Path
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.346 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.346 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ display_cygpath: false
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:Shell() - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 segment.go:Execute:172 ΓåÆ segment: Path
[DEBUG] 11:22:06.346 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.346 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.346 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ display_cygpath: false
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:Shell() - 0s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 segment.go:Execute:172 ΓåÆ segment: Path
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ mapped_locations_enabled: true
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[DEBUG] 11:22:06.346 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[TRACE] 11:22:06.346 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ mapped_locations_enabled: true
[DEBUG] 11:22:06.346 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:Shell() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ home_icon: ~
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ mapped_locations_enabled: true
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[DEBUG] 11:22:06.346 map.go:KeyValueMap:187 ΓåÆ mapped_locations: map[]
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ windows_registry_icon: ∩Çô
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ home_icon: ~
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:KeyValueMap:187 ΓåÆ mapped_locations: map[]
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ home_icon: ~
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:KeyValueMap:192 ΓåÆ mapped_locations: map[C:/BenLocal/git:\ C:/Work/repos:\ {{ .Env.OneDrive }}:\]
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ gitdir_format: 
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ style: agnoster
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ max_width: 
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_icon: ..
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ gitdir_format: 
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ display_root: false
[DEBUG] 11:22:06.346 map.go:String:81 ΓåÆ style: full
[DEBUG] 11:22:06.346 map.go:StringArray:199 ΓåÆ cycle: []
[DEBUG] 11:22:06.346 map.go:StringArray:199 ΓåÆ cycle: []
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_template: 
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_template: 
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_icon: \
[DEBUG] 11:22:06.346 text.go:patchTemplate:149 ΓåÆ {{ (call .Getenv "OneDrive") }}
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ cycle_folder_separator: false
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ edge_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_icon: \
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ cycle_folder_separator: false
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_format: %s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ edge_format: %s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:StackCount() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ left_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ right_format: %s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:StackCount() - 0s
[DEBUG] 11:22:06.346 terminal.go:Getenv:64 ΓåÆ C:\Users\ben.allred\OneDrive
[TRACE] 11:22:06.346 terminal.go:Getenv(OneDrive) - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 separator.go:Separator() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ gitdir_format: 
[DEBUG] 11:22:06.346 map.go:String:81 ΓåÆ style: full
[DEBUG] 11:22:06.346 map.go:StringArray:199 ΓåÆ cycle: []
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_template: 
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_separator_icon: \
[DEBUG] 11:22:06.346 map.go:Bool:112 ΓåÆ cycle_folder_separator: false
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ folder_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ edge_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ left_format: %s
[DEBUG] 11:22:06.346 map.go:String:77 ΓåÆ right_format: %s
[TRACE] 11:22:06.346 terminal.go:Flags() - 0s
[TRACE] 11:22:06.346 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.346 terminal.go:StackCount() - 0s
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:263 ΓåÆ current user is member of S-1-1-0
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:267 ΓåÆ ACCESS_DENIED_ACE_TYPE
[TRACE] 11:22:06.348 terminal_windows.go:DirIsWritable() - 2.229ms
[DEBUG] 11:22:06.348 text.go:patchTemplate:149 ΓåÆ {{ if regexMatch (replace "/" "\\\\" "C:/BenLocal/git(/.+)?$") .Data.Path }} git {{ end }}
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:263 ΓåÆ current user is member of S-1-1-0
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:267 ΓåÆ ACCESS_DENIED_ACE_TYPE
[TRACE] 11:22:06.348 terminal_windows.go:DirIsWritable() - 2.229ms
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:263 ΓåÆ current user is member of S-1-1-0
[DEBUG] 11:22:06.348 win32_windows.go:isWriteable:267 ΓåÆ ACCESS_DENIED_ACE_TYPE
[TRACE] 11:22:06.348 terminal_windows.go:DirIsWritable() - 2.229ms
[DEBUG] 11:22:06.348 text.go:patchTemplate:149 ΓåÆ {{ if regexMatch (replace "/" "\\\\" "C:/Work/repos(/.+)?$") .Data.Path }} {{- if (call .Getenv "WorkReposLabel") }} {{ (call .Getenv "WorkReposLabel") }} {{ else }} repos {{ end -}} {{ end }}
[DEBUG] 11:22:06.348 text.go:patchTemplate:149 ΓåÆ {{ if regexMatch (replace "/" "\\\\" "~/OneDrive(/.+)?$") .Data.Path }} OneDrive {{ end }}
[DEBUG] 11:22:06.348 text.go:patchTemplate:149 ΓåÆ  {{ regexReplaceAll "^\\\\(.+)" .Data.Path "$1" }} 
[DEBUG] 11:22:06.348 text.go:patchTemplate:149 ΓåÆ  {{ if .Data.StackCount }}{{ .Data.StackCount }}{{ end }} 
[TRACE] 11:22:06.348 terminal.go:Shell() - 0s
[TRACE] 11:22:06.348 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.348 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.348 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.348 segment.go:Execute:172 ΓåÆ segment: Dotnet
[DEBUG] 11:22:06.348 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.348 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.348 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.348 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.348 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.348 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.348 segment.go:Execute:172 ΓåÆ segment: Git
[DEBUG] 11:22:06.348 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.348 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.348 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.348 map.go:StringArray:199 ΓåÆ extensions: [*.cs *.csx *.vb *.sln *.slnx *.slnf *.csproj *.vbproj *.fs *.fsx *.fsproj global.json]
[DEBUG] 11:22:06.348 map.go:StringArray:199 ΓåÆ folders: []
[DEBUG] 11:22:06.348 map.go:Bool:112 ΓåÆ home_enabled: false
[DEBUG] 11:22:06.348 map.go:Bool:112 ΓåÆ disable_with_jj: false
[DEBUG] 11:22:06.348 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.348 map.go:String:77 ΓåÆ display_mode: files
[TRACE] 11:22:06.348 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.348 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.348 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.348 segment.go:Execute:172 ΓåÆ segment: Terraform
[DEBUG] 11:22:06.348 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.348 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.348 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.348 map.go:String:77 ΓåÆ command: terraform
[DEBUG] 11:22:06.348 map.go:Bool:120 ΓåÆ fetch_version: true
[TRACE] 11:22:06.348 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.348 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.348 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.348 segment.go:Execute:172 ΓåÆ segment: Battery
[DEBUG] 11:22:06.348 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.348 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.348 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.348 terminal_windows.go:IsWsl() - 0s
[TRACE] 11:22:06.349 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.349 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.349 segment.go:Execute:172 ΓåÆ segment: Text
[DEBUG] 11:22:06.349 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.349 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.349 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.349 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.349 terminal.go:Flags() - 0s
[TRACE] 11:22:06.349 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.349 segment.go:Execute:172 ΓåÆ segment: FileTypes
[DEBUG] 11:22:06.349 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.349 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.349 segment.go:isToggled:293 ΓåÆ no toggles found
[TRACE] 11:22:06.349 terminal.go:Flags() - 0s
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.349 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.349 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.349 segment.go:Execute:172 ΓåÆ segment: Angular
[DEBUG] 11:22:06.349 segment.go:Execute:172 ΓåÆ segment: Node
[DEBUG] 11:22:06.349 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.349 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.349 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.349 store.go:]:154 ΓåÆ (session) found entry: toggle_cache - map[]
[TRACE] 11:22:06.349 store.go:](session toggle_cache) - 0s
[DEBUG] 11:22:06.349 segment.go:isToggled:293 ΓåÆ no toggles found
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ extensions: [angular.json]
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ folders: []
[DEBUG] 11:22:06.349 map.go:Bool:112 ΓåÆ home_enabled: false
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.349 map.go:String:77 ΓåÆ display_mode: files
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ folders: []
[DEBUG] 11:22:06.349 map.go:Bool:112 ΓåÆ home_enabled: false
[DEBUG] 11:22:06.349 home.go:12 ΓåÆ C:\Users\ben.allred
[DEBUG] 11:22:06.349 map.go:String:77 ΓåÆ display_mode: files
[DEBUG] 11:22:06.349 map.go:Bool:120 ΓåÆ fetch_package_manager: true
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.cs) - 534.9┬╡s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.csx) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.vb) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.sln) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.slnx) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.slnf) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.csproj) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.vbproj) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.fs) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.fsx) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.fsproj) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(global.json) - 0s
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ tooling: []
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(angular.json) - 566.4┬╡s
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ tooling: [angular]
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(pnpm-lock.yaml) - 566.4┬╡s
[DEBUG] 11:22:06.349 terminal.go:ResolveSymlink:195 ΓåÆ C:\Users\ben.allred\OneDrive\Work\work-configs
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(yarn.lock) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(bun.lockb) - 0s
[TRACE] 11:22:06.349 terminal.go:ResolveSymlink(C:\Users\ben.allred\OneDrive\Work\work-configs) - 1.1013ms
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(bun.lock) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(package-lock.json) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(package.json) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.js) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(*.ts) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(package.json) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(.nvmrc) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 0s
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 0s
[TRACE] 11:22:06.349 terminal.go:HasParentFilePath(.git) - 1.1013ms
[DEBUG] 11:22:06.349 terminal.go:HasFilesInDir:151 ΓåÆ false
[DEBUG] 11:22:06.349 map.go:Bool:112 ΓåÆ fetch_bare_info: false
[TRACE] 11:22:06.349 terminal.go:HasFilesInDir(.vue) - 0s
[DEBUG] 11:22:06.349 map.go:StringArray:199 ΓåÆ tooling: [node]
[TRACE] 11:22:06.349 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.352 terminal.go:CommandPath:304 ΓåÆ C:\Program Files\Git\cmd\git.exe
[TRACE] 11:22:06.352 terminal.go:CommandPath(git.exe) - 2.9981ms
[TRACE] 11:22:06.352 terminal.go:HasCommand(git.exe) - 2.9981ms
[DEBUG] 11:22:06.352 home.go:12 ΓåÆ C:\Users\ben.allred
[TRACE] 11:22:06.352 terminal.go:GOOS() - 0s
[TRACE] 11:22:06.352 terminal.go:GOOS() - 0s
[DEBUG] 11:22:06.352 map.go:Bool:112 ΓåÆ fetch_user: false
[DEBUG] 11:22:06.352 map.go:String:81 ΓåÆ source: pwsh
[DEBUG] 11:22:06.352 terminal.go:Getenv:64 ΓåÆ NO DATA
[TRACE] 11:22:06.352 terminal.go:Getenv(POSH_GIT_STATUS) - 0s
[ERROR] 11:22:06.352 posh_git.go:hasPoshGitStatus:51 ΓåÆ POSH_GIT_STATUS environment variable not set, do you have the posh-git module installed?
[DEBUG] 11:22:06.352 map.go:Bool:120 ΓåÆ fetch_status: false
[DEBUG] 11:22:06.353 terminal.go:FileContent:212 Γåô
    ref: refs/heads/master
    
[TRACE] 11:22:06.353 terminal.go:FileContent(C:\Users\ben.allred\OneDrive\Work\work-configs\.git/HEAD) - 1.1445ms
[DEBUG] 11:22:06.353 git.go:updateHEADReference:974 ΓåÆ HEADRef: ref: refs/heads/master
[DEBUG] 11:22:06.353 git.go:updateHEADReference:999 ΓåÆ current HEAD is a branch: master
[DEBUG] 11:22:06.353 map.go:String:81 ΓåÆ branch_icon: <p:yellow>≤░ÿ¼</> 
[DEBUG] 11:22:06.353 map.go:KeyValueMap:187 ΓåÆ mapped_branches: map[]
[DEBUG] 11:22:06.353 map.go:String:77 ΓåÆ branch_template: 
[DEBUG] 11:22:06.353 map.go:Bool:112 ΓåÆ fetch_upstream_icon: false
[DEBUG] 11:22:06.359 terminal.go:CommandPath:304 ΓåÆ C:\Users\ben.allred\AppData\Local\Microsoft\WinGet\Links\terraform.exe
[TRACE] 11:22:06.359 terminal.go:CommandPath(terraform) - 11.3967ms
[TRACE] 11:22:06.359 terminal.go:HasCommand(terraform) - 11.3967ms
[DEBUG] 11:22:06.359 terminal.go:HasFolder:180 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFolder(C:\Users\ben.allred\OneDrive\Work\work-configs\.terraform) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(.tf) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(.tfplan) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(.tfstate) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(versions.tf) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(main.tf) - 0s
[DEBUG] 11:22:06.359 terminal.go:HasFilesInDir:151 ΓåÆ false
[TRACE] 11:22:06.359 terminal.go:HasFilesInDir(terraform.tfstate) - 0s
[DEBUG] 11:22:06.359 text.go:patchTemplate:149 ΓåÆ {{ (call .Getenv "omp_files_readme") }}{{ (call .Getenv "omp_files_dockercompose") }}
[DEBUG] 11:22:06.359 terminal.go:Getenv:64 ΓåÆ NO DATA
[TRACE] 11:22:06.359 terminal.go:Getenv(omp_files_readme) - 0s
[DEBUG] 11:22:06.359 terminal.go:Getenv:64 ΓåÆ NO DATA
[TRACE] 11:22:06.359 terminal.go:Getenv(omp_files_dockercompose) - 0s
[TRACE] 11:22:06.370 terminal_windows_nix.go:BatteryState() - 21.5046ms
[DEBUG] 11:22:06.370 map.go:String:81 ΓåÆ charged_icon: ≤▒ƒó
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if and (not .Data.Error) (lt .Data.Percentage 100) }}{{ .Data.Icon }}<p:console:foreground>{{ .Data.Percentage }}%</>{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "Dotnet" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "Node" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "Angular" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "Terraform" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "FileTypes" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  {{ if .Segments.Contains "Battery" }}εé▒{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ  <p:console:foreground>{{ .Data.UpstreamIcon }}</>{{ .Data.HEAD }}{{ if .Data.BranchStatus }} {{ .Data.BranchStatus }}{{ end }}{{ if .Data.Working.Changed }}<p:git:behind> {{ .Data.Working.String }}</>{{ end }}{{ if and (.Data.Staging.Changed) (.Data.Working.Changed) }}{{ end }}{{ if .Data.Staging.Changed }}<p:git:ahead> {{ .Data.Staging.String }}</>{{ end }}<p:console:foreground> εé▒</>
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if and (gt .Data.Ahead 0) (gt .Data.Behind 0) }}p:git:ahead_and_behind{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if gt .Data.Ahead 0 }}p:git:ahead{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if gt .Data.Behind 0 }}p:git:behind{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if and (gt .Data.Ahead 0) (gt .Data.Behind 0) }}p:git:ahead_and_behind{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if gt .Data.Ahead 0 }}p:git:ahead{{ end }}
[DEBUG] 11:22:06.370 text.go:patchTemplate:149 ΓåÆ {{ if gt .Data.Behind 0 }}p:git:behind{{ end }}
[TRACE] 11:22:06.370 terminal.go:Shell() - 0s
[TRACE] 11:22:06.370 terminal.go:Flags() - 0s
[TRACE] 11:22:06.370 terminal.go:Shell() - 0s
[TRACE] 11:22:06.370 path.go:Path() - 0s
```

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@ src/keys
 *.prof
 *.wixpdb
 packages/msi/Microsoft.Trusted.Signing.Client
+.claude
 
 # Created by https://www.toptal.com/developers/gitignore/api/node,go,visualstudiocode
 # Edit at https://www.toptal.com/developers/gitignore?templates=node,go,visualstudiocode

--- a/src/cache/cache.go
+++ b/src/cache/cache.go
@@ -23,6 +23,7 @@ const (
 	PROMPTCOUNTCACHE = "prompt_count_cache"
 	ENGINECACHE      = "engine_cache"
 	FONTLISTCACHE    = "font_list_cache"
+	CLAUDECACHE      = "claude_cache"
 )
 
 type Entry[T any] struct {

--- a/src/cli/claude.go
+++ b/src/cli/claude.go
@@ -61,8 +61,8 @@ Example usage in Claude Code settings:
 
 		var cfg *config.Config
 
-		cfg, err = config.Parse(configFlag)
-		if err != nil {
+		cfg, errCode := config.Parse(configFlag)
+		if errCode != 0 {
 			cfg = config.Claude()
 		}
 

--- a/src/cli/claude.go
+++ b/src/cli/claude.go
@@ -1,6 +1,7 @@
 package cli
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
 	"os"
@@ -10,6 +11,7 @@ import (
 	"github.com/jandedobbeleer/oh-my-posh/src/log"
 	"github.com/jandedobbeleer/oh-my-posh/src/prompt"
 	"github.com/jandedobbeleer/oh-my-posh/src/runtime"
+	"github.com/jandedobbeleer/oh-my-posh/src/segments"
 	"github.com/jandedobbeleer/oh-my-posh/src/shell"
 	"github.com/jandedobbeleer/oh-my-posh/src/template"
 	"github.com/jandedobbeleer/oh-my-posh/src/terminal"
@@ -44,17 +46,13 @@ Example usage in Claude Code settings:
 
 		log.Debugf("received data from stdin: %s", string(stdinData))
 
-		// Store the JSON in an environment variable for the Claude segment to read
-		if len(stdinData) > 0 {
-			os.Setenv("POSH_CLAUDE_STATUS", string(stdinData))
-		}
+		// Process Claude data and initialize cache
+		processClaudeData(stdinData)
 
 		flags := &runtime.Flags{
 			ConfigPath: configFlag,
 			Shell:      shell.CLAUDE,
 		}
-
-		cache.Init(shell.CLAUDE, cache.Persist, cache.NoSession)
 
 		env := &runtime.Terminal{}
 		env.Init(flags)
@@ -76,9 +74,44 @@ Example usage in Claude Code settings:
 			Env:    env,
 		}
 
+		defer func() {
+			template.SaveCache()
+			cache.Close()
+		}()
+
 		result := eng.Status()
 		fmt.Print(result)
 	},
+}
+
+// processClaudeData handles parsing and caching of Claude JSON data
+func processClaudeData(stdinData []byte) {
+	if len(stdinData) == 0 {
+		cache.Init(shell.CLAUDE, cache.Persist, cache.NoSession)
+		return
+	}
+
+	var claudeData segments.Claude
+	if err := json.Unmarshal(stdinData, &claudeData); err != nil {
+		log.Error(err)
+		cache.Init(shell.CLAUDE, cache.Persist, cache.NoSession)
+		return
+	}
+
+	log.Debugf("parsed Claude data: session_id=%s, model=%s", claudeData.SessionID, claudeData.Model.DisplayName)
+
+	// Set the session ID from Claude data if available
+	if claudeData.SessionID != "" {
+		os.Setenv("POSH_SESSION_ID", claudeData.SessionID)
+		log.Debugf("set POSH_SESSION_ID to: %s", claudeData.SessionID)
+	}
+
+	// Initialize cache first so we can store the data
+	cache.Init(shell.CLAUDE, cache.Persist)
+
+	// Store the parsed data in session cache
+	cache.Set(cache.Session, cache.CLAUDECACHE, claudeData, cache.INFINITE)
+	log.Debug("stored Claude data in session cache")
 }
 
 func init() {

--- a/src/cli/claude.go
+++ b/src/cli/claude.go
@@ -59,8 +59,8 @@ Example usage in Claude Code settings:
 
 		var cfg *config.Config
 
-		cfg, errCode := config.Parse(configFlag)
-		if errCode != 0 {
+		cfg, err = config.Parse(configFlag)
+		if err != nil {
 			cfg = config.Claude()
 		}
 

--- a/src/cli/claude.go
+++ b/src/cli/claude.go
@@ -91,7 +91,7 @@ func processClaudeData(stdinData []byte) {
 		return
 	}
 
-	var claudeData segments.Claude
+	var claudeData segments.ClaudeData
 	if err := json.Unmarshal(stdinData, &claudeData); err != nil {
 		log.Error(err)
 		cache.Init(shell.CLAUDE, cache.Persist, cache.NoSession)

--- a/src/config/default.go
+++ b/src/config/default.go
@@ -8,28 +8,13 @@ import (
 	"github.com/jandedobbeleer/oh-my-posh/src/segments/options"
 )
 
-func Default(errorCode int) *Config {
+func Default(configError error) *Config {
 	exitBackgroundTemplate := "{{ if gt .Code 0 }}p:red{{ end }}"
 	exitTemplate := " {{ if gt .Code 0 }}\uf00d{{ else }}\uf00c{{ end }} "
 
-	if errorCode != 0 {
+	if configError != nil && configError != ErrNoConfig {
 		exitBackgroundTemplate = "p:red"
-
-		// Display specific error message based on error code
-		switch errorCode {
-		case ErrorFileNotFound:
-			exitTemplate = " FILE NOT FOUND "
-		case ErrorInvalidExtension:
-			exitTemplate = " INVALID EXTENSION "
-		case ErrorInvalidTheme:
-			exitTemplate = " INVALID THEME "
-		case ErrorURLFetch:
-			exitTemplate = " URL FETCH FAILED "
-		case ErrorParse:
-			exitTemplate = " PARSE ERROR "
-		default:
-			exitTemplate = " CONFIG ERROR "
-		}
+		exitTemplate = configError.Error()
 	}
 
 	cfg := &Config{

--- a/src/config/default.go
+++ b/src/config/default.go
@@ -8,13 +8,28 @@ import (
 	"github.com/jandedobbeleer/oh-my-posh/src/segments/options"
 )
 
-func Default(warning bool) *Config {
+func Default(errorCode int) *Config {
 	exitBackgroundTemplate := "{{ if gt .Code 0 }}p:red{{ end }}"
 	exitTemplate := " {{ if gt .Code 0 }}\uf00d{{ else }}\uf00c{{ end }} "
 
-	if warning {
+	if errorCode != 0 {
 		exitBackgroundTemplate = "p:red"
-		exitTemplate = " CONFIG ERROR "
+
+		// Display specific error message based on error code
+		switch errorCode {
+		case ErrorFileNotFound:
+			exitTemplate = " FILE NOT FOUND "
+		case ErrorInvalidExtension:
+			exitTemplate = " INVALID EXTENSION "
+		case ErrorInvalidTheme:
+			exitTemplate = " INVALID THEME "
+		case ErrorURLFetch:
+			exitTemplate = " URL FETCH FAILED "
+		case ErrorParse:
+			exitTemplate = " PARSE ERROR "
+		default:
+			exitTemplate = " CONFIG ERROR "
+		}
 	}
 
 	cfg := &Config{

--- a/src/config/load.go
+++ b/src/config/load.go
@@ -27,16 +27,22 @@ import (
 // custom no config error
 var ErrNoConfig = errors.New("no config file specified")
 
+// Error codes for config validation (0 = no error)
+const (
+	ErrorNoConfig         = 0 // No error
+	ErrorFileNotFound     = 1 // Config file doesn't exist
+	ErrorInvalidExtension = 2 // Unsupported file extension
+	ErrorInvalidTheme     = 3 // Theme name doesn't exist
+	ErrorURLFetch         = 4 // Failed to fetch from URL
+	ErrorParse            = 5 // Failed to parse config content
+)
+
 func Load(configFile string) *Config {
 	defer log.Trace(time.Now())
 
-	if configFile == "" {
-		return Default(false)
-	}
-
-	cfg, err := Parse(configFile)
-	if err != nil {
-		cfg = Default(true)
+	cfg, errCode := Parse(configFile)
+	if errCode != 0 {
+		cfg = Default(errCode)
 	}
 
 	return cfg
@@ -79,8 +85,13 @@ type hashWriter interface {
 	Write(p []byte) (n int, err error)
 }
 
-func Parse(configFile string) (*Config, error) {
+func Parse(configFile string) (*Config, int) {
 	defer log.Trace(time.Now())
+
+	if configFile == "" {
+		log.Debug("no config file specified")
+		return Default(ErrorNoConfig), ErrorNoConfig
+	}
 
 	configFile = resolveConfigLocation(configFile)
 
@@ -92,25 +103,25 @@ func Parse(configFile string) (*Config, error) {
 
 	h := fnv.New64a()
 
-	cfg, err := read(configFile, h)
-	if err != nil {
-		log.Error(err)
-		return nil, err
+	cfg, errCode := read(configFile, h)
+	if errCode != 0 {
+		log.Errorf("failed to read config: %s", configFile)
+		return nil, errCode
 	}
 
 	parentFolder := filepath.Dir(configFile)
 
 	for cfg.Extends != "" {
 		cfg.Extends = resolvePath(cfg.Extends, parentFolder)
-		base, err := read(cfg.Extends, h)
-		if err != nil {
-			log.Error(err)
+		base, errCode := read(cfg.Extends, h)
+		if errCode != 0 {
+			log.Errorf("failed to read extended config: %s", cfg.Extends)
 			break
 		}
 
 		configDSC.Add(cfg.Extends)
 
-		err = base.merge(cfg)
+		err := base.merge(cfg)
 		if err != nil {
 			log.Error(err)
 			break
@@ -140,7 +151,7 @@ func Parse(configFile string) (*Config, error) {
 		cfg.Upgrade.Interval = cache.ONEWEEK
 	}
 
-	return cfg, nil
+	return cfg, 0
 }
 
 func resolvePath(configFile, parentFolder string) string {
@@ -161,12 +172,12 @@ func resolvePath(configFile, parentFolder string) string {
 	return filepath.Join(parentFolder, configFile)
 }
 
-func read(configFile string, h hashWriter) (*Config, error) {
+func read(configFile string, h hashWriter) (*Config, int) {
 	defer log.Trace(time.Now())
 
 	if configFile == "" {
 		log.Debug("no config file specified, using default")
-		return Default(false), nil
+		return Default(0), 0
 	}
 
 	var cfg Config
@@ -175,13 +186,24 @@ func read(configFile string, h hashWriter) (*Config, error) {
 
 	data, err := getData(configFile)
 	if err != nil {
-		return nil, err
+		// Determine the type of error
+		if strings.HasPrefix(configFile, "https://") {
+			log.Errorf("failed to fetch config from URL: %v", err)
+			return nil, ErrorURLFetch
+		}
+		if errors.Is(err, os.ErrNotExist) {
+			log.Errorf("config file not found: %v", err)
+			return nil, ErrorFileNotFound
+		}
+		log.Errorf("failed to read config: %v", err)
+		return nil, ErrorFileNotFound
 	}
 
+	var parseErr error
 	switch cfg.Format {
 	case YAML, YML:
 		cfg.Format = YAML
-		err = yaml.Unmarshal(data, &cfg)
+		parseErr = yaml.Unmarshal(data, &cfg)
 	case JSONC, JSON:
 		cfg.Format = JSON
 
@@ -189,16 +211,18 @@ func read(configFile string, h hashWriter) (*Config, error) {
 		data = []byte(str)
 
 		decoder := json.NewDecoder(bytes.NewReader(data))
-		err = decoder.Decode(&cfg)
+		parseErr = decoder.Decode(&cfg)
 	case TOML, TML:
 		cfg.Format = TOML
-		err = toml.Unmarshal(data, &cfg)
+		parseErr = toml.Unmarshal(data, &cfg)
 	default:
-		err = fmt.Errorf("unsupported config file format: %s", cfg.Format)
+		log.Errorf("unsupported config file format: %s", cfg.Format)
+		return nil, ErrorInvalidExtension
 	}
 
-	if err != nil {
-		return nil, err
+	if parseErr != nil {
+		log.Errorf("failed to parse config: %v", parseErr)
+		return nil, ErrorParse
 	}
 
 	_, err = h.Write(data)
@@ -206,7 +230,7 @@ func read(configFile string, h hashWriter) (*Config, error) {
 		log.Error(err)
 	}
 
-	return &cfg, nil
+	return &cfg, 0
 }
 
 func getData(configFile string) ([]byte, error) {

--- a/src/config/load.go
+++ b/src/config/load.go
@@ -24,25 +24,30 @@ import (
 	yaml "go.yaml.in/yaml/v3"
 )
 
-// custom no config error
-var ErrNoConfig = errors.New("no config file specified")
+// Custom error types for config validation
+type Error struct {
+	message string
+}
 
-// Error codes for config validation (0 = no error)
-const (
-	ErrorNoConfig         = 0 // No error
-	ErrorFileNotFound     = 1 // Config file doesn't exist
-	ErrorInvalidExtension = 2 // Unsupported file extension
-	ErrorInvalidTheme     = 3 // Theme name doesn't exist
-	ErrorURLFetch         = 4 // Failed to fetch from URL
-	ErrorParse            = 5 // Failed to parse config content
+func (e Error) Error() string {
+	return fmt.Sprintf(" %s ", e.message)
+}
+
+var (
+	ErrFileNotFound     = Error{"CONFIG NOT FOUND"}
+	ErrInvalidExtension = Error{"INVALID CONFIG EXTENSION"}
+	ErrInvalidTheme     = Error{"INVALID CONFIG THEME"}
+	ErrURLFetch         = Error{"CONFIG URL FETCH FAILED"}
+	ErrParse            = Error{"CONFIG PARSE ERROR"}
+	ErrNoConfig         = Error{"NO CONFIG"}
 )
 
 func Load(configFile string) *Config {
 	defer log.Trace(time.Now())
 
-	cfg, errCode := Parse(configFile)
-	if errCode != 0 {
-		cfg = Default(errCode)
+	cfg, err := Parse(configFile)
+	if err != nil {
+		cfg = Default(err)
 	}
 
 	return cfg
@@ -85,12 +90,12 @@ type hashWriter interface {
 	Write(p []byte) (n int, err error)
 }
 
-func Parse(configFile string) (*Config, int) {
+func Parse(configFile string) (*Config, error) {
 	defer log.Trace(time.Now())
 
 	if configFile == "" {
 		log.Debug("no config file specified")
-		return Default(ErrorNoConfig), ErrorNoConfig
+		return nil, ErrNoConfig
 	}
 
 	configFile = resolveConfigLocation(configFile)
@@ -103,25 +108,25 @@ func Parse(configFile string) (*Config, int) {
 
 	h := fnv.New64a()
 
-	cfg, errCode := read(configFile, h)
-	if errCode != 0 {
+	cfg, err := read(configFile, h)
+	if err != nil {
 		log.Errorf("failed to read config: %s", configFile)
-		return nil, errCode
+		return nil, err
 	}
 
 	parentFolder := filepath.Dir(configFile)
 
 	for cfg.Extends != "" {
 		cfg.Extends = resolvePath(cfg.Extends, parentFolder)
-		base, errCode := read(cfg.Extends, h)
-		if errCode != 0 {
+		base, err := read(cfg.Extends, h)
+		if err != nil {
 			log.Errorf("failed to read extended config: %s", cfg.Extends)
 			break
 		}
 
 		configDSC.Add(cfg.Extends)
 
-		err := base.merge(cfg)
+		err = base.merge(cfg)
 		if err != nil {
 			log.Error(err)
 			break
@@ -151,7 +156,7 @@ func Parse(configFile string) (*Config, int) {
 		cfg.Upgrade.Interval = cache.ONEWEEK
 	}
 
-	return cfg, 0
+	return cfg, nil
 }
 
 func resolvePath(configFile, parentFolder string) string {
@@ -172,12 +177,12 @@ func resolvePath(configFile, parentFolder string) string {
 	return filepath.Join(parentFolder, configFile)
 }
 
-func read(configFile string, h hashWriter) (*Config, int) {
+func read(configFile string, h hashWriter) (*Config, error) {
 	defer log.Trace(time.Now())
 
 	if configFile == "" {
 		log.Debug("no config file specified, using default")
-		return Default(0), 0
+		return Default(nil), nil
 	}
 
 	var cfg Config
@@ -189,14 +194,14 @@ func read(configFile string, h hashWriter) (*Config, int) {
 		// Determine the type of error
 		if strings.HasPrefix(configFile, "https://") {
 			log.Errorf("failed to fetch config from URL: %v", err)
-			return nil, ErrorURLFetch
+			return nil, ErrURLFetch
 		}
 		if errors.Is(err, os.ErrNotExist) {
 			log.Errorf("config file not found: %v", err)
-			return nil, ErrorFileNotFound
+			return nil, ErrFileNotFound
 		}
 		log.Errorf("failed to read config: %v", err)
-		return nil, ErrorFileNotFound
+		return nil, ErrFileNotFound
 	}
 
 	var parseErr error
@@ -217,12 +222,12 @@ func read(configFile string, h hashWriter) (*Config, int) {
 		parseErr = toml.Unmarshal(data, &cfg)
 	default:
 		log.Errorf("unsupported config file format: %s", cfg.Format)
-		return nil, ErrorInvalidExtension
+		return nil, ErrInvalidExtension
 	}
 
 	if parseErr != nil {
 		log.Errorf("failed to parse config: %v", parseErr)
-		return nil, ErrorParse
+		return nil, ErrParse
 	}
 
 	_, err = h.Write(data)
@@ -230,7 +235,7 @@ func read(configFile string, h hashWriter) (*Config, int) {
 		log.Error(err)
 	}
 
-	return &cfg, 0
+	return &cfg, nil
 }
 
 func getData(configFile string) ([]byte, error) {

--- a/src/config/segment_types.go
+++ b/src/config/segment_types.go
@@ -43,6 +43,7 @@ func init() {
 	gob.Register(&segments.Cf{})
 	gob.Register(&segments.CfTarget{})
 	gob.Register(&segments.Claude{})
+	gob.Register(&segments.ClaudeData{})
 	gob.Register(&segments.Clojure{})
 	gob.Register(&segments.Cmake{})
 	gob.Register(&segments.Connection{})

--- a/src/segments/claude.go
+++ b/src/segments/claude.go
@@ -1,9 +1,9 @@
 package segments
 
 import (
-	"encoding/json"
 	"fmt"
 
+	"github.com/jandedobbeleer/oh-my-posh/src/cache"
 	"github.com/jandedobbeleer/oh-my-posh/src/log"
 	"github.com/jandedobbeleer/oh-my-posh/src/text"
 )
@@ -52,9 +52,8 @@ type ClaudeCurrentUsage struct {
 }
 
 const (
-	poshClaudeStatusEnv = "POSH_CLAUDE_STATUS"
-	thousand            = 1000.0
-	million             = 1000000.0
+	thousand = 1000.0
+	million  = 1000000.0
 )
 
 func (c *Claude) Template() string {
@@ -64,22 +63,21 @@ func (c *Claude) Template() string {
 func (c *Claude) Enabled() bool {
 	log.Debug("claude segment: checking if enabled")
 
-	claudeStatus := c.env.Getenv(poshClaudeStatusEnv)
-	if len(claudeStatus) == 0 {
-		log.Debug("claude segment: POSH_CLAUDE_STATUS is empty")
+	// Try to get Claude data from session cache
+	claudeData, found := cache.Get[Claude](cache.Session, cache.CLAUDECACHE)
+	if !found {
+		log.Debug("claude segment: no Claude data found in session cache")
 		return false
 	}
 
-	log.Debugf("claude segment: parsing JSON (%d bytes)", len(claudeStatus))
+	log.Debug("claude segment: found Claude data in session cache")
+	log.Debugf("claude segment: model=%s, session=%s", claudeData.Model.DisplayName, claudeData.SessionID)
 
-	err := json.Unmarshal([]byte(claudeStatus), c)
-	if err != nil {
-		log.Error(err)
-		return false
-	}
-
-	log.Debug("claude segment: successfully parsed JSON")
-	log.Debugf("claude segment: model=%s, session=%s", c.Model.DisplayName, c.SessionID)
+	// Copy the data to our struct
+	// Preserve the Base struct to avoid nil pointer dereference
+	base := c.Base
+	*c = claudeData
+	c.Base = base
 
 	return true
 }

--- a/src/segments/claude.go
+++ b/src/segments/claude.go
@@ -11,7 +11,11 @@ import (
 // Claude segment displays Claude Code session information
 type Claude struct {
 	Base
+	ClaudeData
+}
 
+// ClaudeData represents the parsed Claude JSON data
+type ClaudeData struct {
 	SessionID     string              `json:"session_id"`
 	Model         ClaudeModel         `json:"model"`
 	Workspace     ClaudeWorkspace     `json:"workspace"`
@@ -64,7 +68,7 @@ func (c *Claude) Enabled() bool {
 	log.Debug("claude segment: checking if enabled")
 
 	// Try to get Claude data from session cache
-	claudeData, found := cache.Get[Claude](cache.Session, cache.CLAUDECACHE)
+	claudeData, found := cache.Get[ClaudeData](cache.Session, cache.CLAUDECACHE)
 	if !found {
 		log.Debug("claude segment: no Claude data found in session cache")
 		return false
@@ -73,11 +77,8 @@ func (c *Claude) Enabled() bool {
 	log.Debug("claude segment: found Claude data in session cache")
 	log.Debugf("claude segment: model=%s, session=%s", claudeData.Model.DisplayName, claudeData.SessionID)
 
-	// Copy the data to our struct
-	// Preserve the Base struct to avoid nil pointer dereference
-	base := c.Base
-	*c = claudeData
-	c.Base = base
+	// Copy the data to our embedded struct
+	c.ClaudeData = claudeData
 
 	return true
 }

--- a/src/segments/claude_test.go
+++ b/src/segments/claude_test.go
@@ -3,6 +3,7 @@ package segments
 import (
 	"testing"
 
+	"github.com/jandedobbeleer/oh-my-posh/src/cache"
 	"github.com/jandedobbeleer/oh-my-posh/src/runtime/mock"
 	"github.com/jandedobbeleer/oh-my-posh/src/segments/options"
 	"github.com/jandedobbeleer/oh-my-posh/src/text"
@@ -13,67 +14,60 @@ import (
 func TestClaudeSegment(t *testing.T) {
 	cases := []struct {
 		Case            string
-		ClaudeStatus    string
+		ClaudeData      *Claude
 		ExpectedModel   string
 		ExpectedSession string
 		ExpectedEnabled bool
 	}{
 		{
-			Case:            "Empty environment variable",
-			ClaudeStatus:    "",
+			Case:            "No cache data",
+			ClaudeData:      nil,
 			ExpectedEnabled: false,
 		},
 		{
-			Case:            "Invalid JSON",
-			ClaudeStatus:    "invalid json",
-			ExpectedEnabled: false,
-		},
-		{
-			Case: "Valid JSON with all fields",
-			ClaudeStatus: `{
-				"session_id": "abc123",
-				"model": {
-					"id": "claude-opus-4-1",
-					"display_name": "Opus"
+			Case: "Valid cache data with all fields",
+			ClaudeData: &Claude{
+				SessionID: "abc123",
+				Model: ClaudeModel{
+					ID:          "claude-opus-4-1",
+					DisplayName: "Opus",
 				},
-				"workspace": {
-					"current_dir": "/repo/project",
-					"project_dir": "/repo"
+				Workspace: ClaudeWorkspace{
+					CurrentDir: "/repo/project",
+					ProjectDir: "/repo",
 				},
-				"cost": {
-					"total_cost_usd": 0.01,
-					"total_duration_ms": 45000
+				Cost: ClaudeCost{
+					TotalCostUSD:    0.01,
+					TotalDurationMS: 45000,
 				},
-				"context_window": {
-					"total_input_tokens": 15234,
-					"total_output_tokens": 4521,
-					"context_window_size": 200000,
-					"current_usage": {
-						"input_tokens": 8500,
-						"output_tokens": 1200
-					}
-				}
-			}`,
+				ContextWindow: ClaudeContextWindow{
+					TotalInputTokens:  15234,
+					TotalOutputTokens: 4521,
+					ContextWindowSize: 200000,
+					CurrentUsage: ClaudeCurrentUsage{
+						InputTokens:  8500,
+						OutputTokens: 1200,
+					},
+				},
+			},
 			ExpectedEnabled: true,
 			ExpectedModel:   "Opus",
 			ExpectedSession: "abc123",
 		},
 		{
-			Case: "Valid JSON with partial fields",
-			ClaudeStatus: `{
-				"session_id": "xyz789",
-				"model": {
-					"id": "claude-sonnet-3-5",
-					"display_name": "Sonnet 3.5"
+			Case: "Valid cache data with partial fields",
+			ClaudeData: &Claude{
+				SessionID: "xyz789",
+				Model: ClaudeModel{
+					ID:          "claude-sonnet-3-5",
+					DisplayName: "Sonnet 3.5",
 				},
-				"workspace": {},
-				"cost": {},
-				"context_window": {
-					"total_input_tokens": 1000,
-					"total_output_tokens": 500,
-					"context_window_size": 100000
-				}
-			}`,
+				ContextWindow: ClaudeContextWindow{
+					TotalInputTokens:  1000,
+					TotalOutputTokens: 500,
+					ContextWindowSize: 100000,
+				},
+			},
 			ExpectedEnabled: true,
 			ExpectedModel:   "Sonnet 3.5",
 			ExpectedSession: "xyz789",
@@ -81,9 +75,14 @@ func TestClaudeSegment(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		env := new(mock.Environment)
-		env.On("Getenv", poshClaudeStatusEnv).Return(tc.ClaudeStatus)
+		// Setup cache for test
+		if tc.ClaudeData != nil {
+			cache.Set(cache.Session, cache.CLAUDECACHE, *tc.ClaudeData, cache.INFINITE)
+		} else {
+			cache.Delete(cache.Session, cache.CLAUDECACHE)
+		}
 
+		env := new(mock.Environment)
 		claude := &Claude{
 			Base: Base{
 				env:     env,

--- a/src/segments/claude_test.go
+++ b/src/segments/claude_test.go
@@ -14,7 +14,7 @@ import (
 func TestClaudeSegment(t *testing.T) {
 	cases := []struct {
 		Case            string
-		ClaudeData      *Claude
+		ClaudeData      *ClaudeData
 		ExpectedModel   string
 		ExpectedSession string
 		ExpectedEnabled bool
@@ -26,7 +26,7 @@ func TestClaudeSegment(t *testing.T) {
 		},
 		{
 			Case: "Valid cache data with all fields",
-			ClaudeData: &Claude{
+			ClaudeData: &ClaudeData{
 				SessionID: "abc123",
 				Model: ClaudeModel{
 					ID:          "claude-opus-4-1",
@@ -56,7 +56,7 @@ func TestClaudeSegment(t *testing.T) {
 		},
 		{
 			Case: "Valid cache data with partial fields",
-			ClaudeData: &Claude{
+			ClaudeData: &ClaudeData{
 				SessionID: "xyz789",
 				Model: ClaudeModel{
 					ID:          "claude-sonnet-3-5",

--- a/src/segments/git.go
+++ b/src/segments/git.go
@@ -678,9 +678,11 @@ func (g *Git) cleanUpstreamURL(url string) string {
 }
 
 func (g *Git) getUpstreamIcon() string {
+	fallback := g.options.String(GitIcon, "\uE5FB ")
+
 	g.RawUpstreamURL = g.getRemoteURL()
 	if g.RawUpstreamURL == "" {
-		return ""
+		return fallback
 	}
 
 	g.UpstreamURL = g.cleanUpstreamURL(g.RawUpstreamURL)
@@ -705,12 +707,14 @@ func (g *Git) getUpstreamIcon() string {
 		"codecommit":       {CodeCommit, "\uF270"},
 		"codeberg":         {CodebergIcon, "\uF330"},
 	}
+
 	for key, value := range defaults {
 		if strings.Contains(g.UpstreamURL, key) {
 			return g.options.String(value.Icon, value.Default)
 		}
 	}
-	return g.options.String(GitIcon, "\uE5FB ")
+
+	return fallback
 }
 
 func (g *Git) setStatus() {

--- a/src/segments/git_test.go
+++ b/src/segments/git_test.go
@@ -676,7 +676,7 @@ func TestGitUpstream(t *testing.T) {
 		Expected string
 		Upstream string
 	}{
-		{Case: "No upstream", Expected: "", Upstream: ""},
+		{Case: "No upstream", Expected: "G", Upstream: ""},
 		{Case: "SSH url", Expected: "G", Upstream: "ssh://git@git.my.domain:3001/ADIX7/dotconfig.git"},
 		{Case: "Gitea", Expected: "EX", Upstream: "_gitea@src.example.com:user/repo.git"},
 		{Case: "GitHub", Expected: "GH", Upstream: "github.com/test"},

--- a/website/blog/2025-12-28-oh-my-posh-claude-code-integration.md
+++ b/website/blog/2025-12-28-oh-my-posh-claude-code-integration.md
@@ -1,0 +1,126 @@
+---
+date: 2025-12-28
+description: Oh My Posh now integrates with Claude Code's statusline feature to bring beautiful, customizable AI session information right into your terminal prompt.
+tags:
+- claude
+- ai
+- terminal
+- prompt
+- statusline
+- integration
+- customization
+title: "Oh My Posh Meets Claude Code: AI-Powered Terminal Prompts"
+slug: oh-my-posh-claude-code-integration
+authors:
+- name: Jan De Dobbeleer
+  title: Maintainer
+  url: https://github.com/jandedobbeleer
+  image_url: https://avatars.githubusercontent.com/u/2492783?v=4
+---
+
+Terminal customization just got a lot smarter. Oh My Posh now integrates with [Claude Code](https://code.claude.com/)
+through its [statusline functionality](https://code.claude.com/docs/en/statusline), bringing real-time AI session
+information directly into your terminal prompt. This isn't just another segment: it's a bridge between your
+development workflow and AI-powered coding assistance.
+
+## What is Claude Code's statusline?
+
+Claude Code's statusline feature allows you to create custom status displays that appear at the bottom of
+the Claude Code interface, similar to how terminal prompts work in shells. The statusline receives rich JSON
+data about your current AI session via stdin, including:
+
+- **Model information**: Which Claude model you're using (Claude Sonnet, Claude Opus, etc.)
+- **Token usage**: Input/output tokens, context window utilization, and usage percentages
+- **Cost tracking**: Real-time cost calculations and session duration
+- **Workspace context**: Current and project directories
+- **Session metadata**: Unique session IDs and version information
+
+The statusline updates automatically when conversation messages change (throttled to every 300ms max), and
+your command's stdout becomes the status display with full ANSI color support.
+
+### Setting Up the Integration
+
+Configuration is straightforward. Add this to your Claude Code settings:
+
+```json title="~/.claude/settings.json"
+{
+  "statusLine": {
+    "command": "oh-my-posh claude"
+  }
+}
+```
+
+That's it! Oh My Posh will automatically detect when Claude Code provides session data and display the
+relevant information in your prompt.
+
+It's important to note that the `claude` CLI command operates differently from your regular prompt
+configuration. When used as a statusline command, Oh My Posh runs in a special mode that's completely separate
+from your standard terminal prompt. This means you'll likely want to create a dedicated, minimal configuration
+specifically for Claude Code that focuses on displaying AI session information rather than your usual prompt
+elements.
+
+By default, the `oh-my-posh claude` command provides a built-in statusline that shows your current working
+directory, git context, active model name, and context window usage as a visual gauge. To customize this
+display, use the `--config` flag to specify your own theme configuration file that includes a custom claude
+segment tailored to your preferences.
+
+## The Claude Code segment
+
+Oh My Posh's new `claude` segment taps into this statusline data to bring AI session awareness directly
+into your terminal prompt. When you use the `oh-my-posh claude` command as your statusline command in Claude
+Code, you get access to a wealth of session information that can be displayed in your prompt without needing
+to know the technical details.
+
+### Example Configuration
+
+Here's a sample configuration that shows the model name and context usage:
+
+```json
+{
+  "type": "claude",
+  "style": "diamond",
+  "leading_diamond": "\ue0b6",
+  "trailing_diamond": "\ue0b4",
+  "foreground": "#FFFFFF",
+  "background": "#FF6B35",
+  "template": " \udb82\udfc9 {{ .Model.DisplayName }} \uf2d0 {{ .TokenUsagePercent.Gauge }} "
+}
+```
+
+This displays something like: ` 🤖 Claude 4.5 Sonnet  ▰▰▰▱▱ `
+
+The gauge provides instant visual feedback on how much of your context window you've consumed, which is
+crucial for managing long coding sessions.
+
+## The Technical Details
+
+Under the hood, Oh My Posh reads the rich JSON session data that Claude Code provides via stdin when used
+as a statusline command. The segment only activates when Claude Code session data is available, so there's no
+performance impact when you're not using Claude Code.
+
+The JSON structure follows Claude Code's specification, ensuring forward compatibility as new features are
+added to the statusline API. The segment handles missing data gracefully and provides sensible defaults for
+any fields that might be unavailable.
+
+## Getting Started
+
+If you're already using Oh My Posh, adding Claude Code integration is as simple as:
+
+1. Install Claude Code if you haven't already
+2. Add the statusline configuration to your Claude Code settings
+3. Add a claude segment to your Oh My Posh theme
+4. Start a Claude Code session and watch your prompt come alive
+
+For detailed configuration options and all available properties, check out the [complete Claude segment documentation](https://ohmyposh.dev/docs/segments/cli/claude).
+
+## What's Next?
+
+This integration opens up exciting possibilities. Imagine prompts that:
+
+- Change color based on token usage percentage
+- Show different icons for different AI models
+- Display cost warnings when sessions get expensive
+- Integrate with git status to show AI-assisted development context
+
+The foundation is there, and now it's up to the community to build amazing configurations that make
+AI-powered development even more seamless.

